package.json

```diff
@@ -6,6 +6,7 @@
   "workspaces": [
     "packages/api",
     "packages/client",
+    "packages/e2e",
```

Comment:
Looks like this package is omitted in this commit, is this intentional @francois-potato?

Reply:
Typical; I forgot to push the new files!